docs/manual/manual.adoc

```diff
@@ -522,6 +522,31 @@ command-line argument. You can use `--results-arf` to generate a Result DataStre
 $ oscap xccdf eval --benchmark-id benchmark_id --results-arf arf-results.xml scap-ds.xml
 --------------------------------------------------------------------------------------
 
+==== Result STIG Viewer
+
+If you want to import the XCCDF scan results to DISA STIG Viewer but your Rule IDs don't
+match the DISA's ones, you can use the `--stig-viewer` command-line argument along with
+a special reference in your Rules to generate XCCDF result files that can be imported by
+DISA STIG Viewer.
+
+Each rule in the input XCCDF must contain a reference to its STIG Rule ID, and its 
+href attribute must be exactly "http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx".
+
+For example:
+----
+<Rule id="rpm_verify_permissions">
+  ...
+  <reference href="http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx">SV-86473r2_rule</reference>
+  ...
+</Rule>
+----
+
+See link:http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx[DISA STIG Viewer].
```

Comment:
It looks like this link will give more information about how to prepare the contents for `--stig-viewer` output.
Suggestion: `See following link for information on what is DISA STIG Viewer`